keyboards/preonic/keymaps/pitty/keymap.c

```diff
@@ -0,0 +1,236 @@
+ */
+[_RAISE] = LAYOUT_preonic_grid( \
+  KC_F12,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,    KC_F11, \
+  _______, _______, KC_UP,   _______, _______, KC_HOME, KC_PGUP, _______, _______, _______, _______,   _______,  \
+  _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, KC_END,  KC_PGDN, _______, _______, _______, _______,   _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, TO(_GAME), TO(_QWERTY)  \
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_preonic_grid( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+  _______, RESET,   DEBUG,   _______, _______, _______, _______, TERM_ON, TERM_OFF,_______, _______, KC_DEL,  \
+  _______, _______, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, _______,  _______, _______, _______, _______, \
+  _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, TO(_QWERTY)  \
+)
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_startup[][2] = SONG(STARTUP_SOUND); 
+float tone_qwerty[][2] = SONG(QWERTY_SOUND);
+float tone_numpad[][2] = SONG(MARIO_THEME);
+float tone_oneshot[][2] = SONG(MARIO_GAMEOVER);
+float tone_dyn_macro_rec[][2] = SONG(ONE_UP_SOUND);
+float tone_dyn_macro_stop[][2] = SONG(COIN_SOUND);
+float music_scale[][2] = SONG(RICK_ROLL);
+float tone_goodbye[][2] = SONG(TERMINAL_SOUND);
+float tone_goodbye1[][2] = SONG(IMPERIAL_MARCH);
+float tone_goodbye2[][2] = SONG(MARIO_MUSHROOM);
+float tone_goodbye3[][2] = SONG(ZELDA_TREASURE);
+float tone_goodbye4[][2] = SONG(ZELDA_PUZZLE);
+float tone_goodbye5[][2] = SONG(E1M1_DOOM);
+float tone_goodbye6[][2] = SONG(OVERWATCH_THEME);
+float real_imp_mar[][2]    = {
```

Comment:
This may be problematic, as any copyrighted material in the repo may be a problem.

Reply:
Thanks, I have replaced them, sorry for this. Hope it's fine now.